packages/bot-web-ui/.eslintrc.js

```diff
@@ -17,4 +17,51 @@ module.exports = {
             webpack: { config: webpackConfig({}) },
         },
     },
-};
\ No newline at end of file
+    plugins: ['simple-import-sort'],
+    rules: {
+        'simple-import-sort/imports': 'error',
+        'simple-import-sort/exports': 'error',
+    },
+    overrides: [
+        {
+            files: ['**/*.js', '**/*.ts', '**/*.tsx', '**/*.jsx'],
+            rules: {
+                'simple-import-sort/imports': [
+                    'error',
+                    {
+                        groups: [
+                            [
+                                // `react` first, `next` second, then packages starting with a character
+                                '^react$',
+                                '^next',
```

Comment:
what's `next` here?
nextjs?

Reply:
nextjs or nextts. libraries which can start with 'next'. Above we have a comment actually. Probably we can remove it for package `bot-web-ui`

Reply:
removed